atd-vzv/src/App.js

```diff
@@ -2,11 +2,26 @@ import React from "react";
 import { usePath } from "hookrouter";
 import SideDrawer from "./views/nav/SideDrawer";
 import Content from "./views/content/Content";
+import UnderMaintenance from "./views/NotFound/UnderMaintenance";
 import "./App.css";
 
 const App = () => {
   const currentPath = usePath();
   const isMeasuresPath = currentPath === "/measures";
+  const isUnderMaintenance = process.env.REACT_APP_UNDER_MAINTENANCE === "True";
+
+  console.log(
+    "Is under maintenance: ",
+    process.env.REACT_APP_UNDER_MAINTENANCE
+  );
```

Comment:
`isUnderMaintenance` and the console logging can be moved outside the `App` component